standards/documentation.md

```diff
@@ -0,0 +1,65 @@
+# <a name="documentation"></a>Documentation
+
+Good documentation makes a long-lived project easier to understand and maintain. It can help introduce new developers to an unfamiliar codebase, or serve as a reference point for the team coming back to code 6 months later. Documentation may also cover important architectural decisions, complex business logic, or quirks and workarounds of the tech we're working with. Just as we put effort into making tests that last to ensure our application is stable and robust, we also focus on providing meaningful documentation.
+
+A great article on improving Rails-based documentation published by Aaron Sumner can be found here. Many of the practices described below are pulled from his philosophy.
+
+## Start with the README
+Your application’s README is still its gateway, the first thing new developers will likely read when they access its source code. If your app’s README consists of the default given to every new Rails app, it’s time to change it.
+
+Your README should include:
+
+* A brief description of what the application does.
+* How to install dependencies, especially those that are more complex than a simple `bundle install`. For example, if I need to to install Redis or ImageMagick to work on the app, or configure environment variables to talk to external services, let me know up front.
+* How to run the application’s test suite, to verify that setup was successful. Again, note special external dependencies, like PhantomJS.
+* How to generate the docs, to get to a more detailed description of the application’s code.
+
+## Remove generated documentation from source control
+
+The HTML version of your Rails docs will need to be regenerated regularly, in order to keep up with the code they explain. Ignoring the doc/app directory makes sure that stale documentation doesn't stick around from commit to commit, and just makes commits cleaner in general. This is why it’s important to mention how to generate the docs in your README as they won’t be easily accessible, otherwise.
+
+## Write clear documentation
+
+* __Don't assume knowledge of the business__: If you've been working in the application for awhile, the names you've given your classes and methods may seem obvious, but to a newcomer they can be a large hurdle to overcome. Help explain ___why___ things are called what they are.
+* __Do assume knowledge of Rails__: The rule of thumb is to document ___why___, now ___how___. Document under the assumption that the reader can read Ruby code, or at least knows how to search Google or Stack Overflow to learn more about specific methods.
```

Comment:
I am not entirely certain on this one.  While I certainly agree documentation isn't meant to be a tutorial on Rails features, or Ruby code itself.  I often find that the document why, not how, does leave a lot of cases where that advice becomes dogma and doesn't make sense in every case.  I'm unsure if i have a great solution, or better wording here.  Maybe I'm an outlier on this one too.